internal/alloycli/cmd_run.go

```diff
@@ -275,17 +280,28 @@ func (fr *alloyRun) Run(configPath string) error {
 		return err
 	}
 
+	var runtimeFlags []byte
```

Comment:
I was avoiding iterating over the list again when converting to a []byte for setting the content in the zip, but it's never going to be a large enough list that that optimization is very important, so it's refactored for clarity/keeping the implementation details in the implementation rather than the interface.